controllers/fenceagentsremediation_controller.go

```diff
@@ -38,6 +38,8 @@ const (
 	errorMissingParams     = "nodeParameters or sharedParameters or both are missing, and they cannot be empty"
 	errorMissingNodeParams = "node parameter is required, and cannot be empty"
 	SuccessFAResponse      = "Success: Rebooted"
```

Comment:
I think this could be made private

Reply:
ATM it can be. But it might make more sense keep it public and use it in https://github.com/medik8s/fence-agents-remediation/blob/main/test/e2e/far_e2e_test.go#L119, instead of creating another const  `succeesRebootMessage`

Reply:
But we do is it in https://github.com/medik8s/fence-agents-remediation/pull/54/files#diff-b9729ac4458c7637438e3bf558999b35adedcaf4d13af1e3eb752db501fd4997R119